dali/operators/reader/loader/numpy_loader_gpu.h

```diff
@@ -49,6 +49,7 @@ struct NumpyFileWrapperGPU {
   TensorShape<> shape;
   DALIDataType type;
   DALIMeta meta;
+  int source_sample_idx;
```

Comment:
I'd suggest initializing (it with something invalid)
```suggestion
  int source_sample_idx = -1;
```